### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by jesshewitt** [noisedeck/noisedeck-docs#2] [Introductions]
Hi! I'm Jess Hewitt and I'm one of Noisedeck's co-developers, along with Alex Ayars. 

We are both very into generative art and use Noisedeck ourselves just about every day.

I can be found on [Twitter](https://twitter.com/rustysniper1) and I have a crypto art gallery at [Try Showtime](https://tryshowtime.com/rustysniper).

Nice to meet you! 😁

**Comment by geoff-maddock** [comment]
Hey folks.  I'm Geoff aka [Cutups](https://linktr.ee/cutups).   I've been an enthusiast of digital music and visuals for a long time, primarily as a promoter and dj.  Been spending a bit more time playing with different visual tools and coding practices during the pandemic, including [Hydra](https://github.com/ojack/hydra) and more recently Noisedeck.
